### PR TITLE
Bb add test data 154932786: TDL data

### PR DESCRIPTION
**Pull request by breanneboland**
/cmd/generate_test_data/main.go creates three records for the TDL table and prints their IDs in the terminal upon creation.

### CODE REVIEW

File: cmd/generate_test_data/main.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	//DB connection
+	pop.AddLookupPaths(*config)
+	dbConnection, err := pop.Connect(*env)
+	if err != nil {
+		log.Panic(err)
+	}
+
+	// Add three TDL table records
+	tdl1 := models.TrafficDistributionList{
+		SourceRateArea:    "california",
```
Comment (jim) [inline cmd/generate_test_data/main.go]: 🏆 for using realistic values!

File: cmd/generate_test_data/main.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"fmt"
+	"github.com/markbates/pop"
+	"github.com/transcom/mymove/pkg/models"
+	"log"
+	// "github.com/satori/go.uuid"
```
Comment (akostibas) [inline cmd/generate_test_data/main.go]: kill stray comments

File: cmd/generate_test_data/main.go (hunk 1 above)
Comment (akostibas) [reply cmd/generate_test_data/main.go]: There are actually more realistic values that we can extract from the PDFs referenced in [How TSPs are awarded shipments](https://docs.google.com/document/d/1LL776HsYZM-54bYe2RZSZtsBLNuLrEvDad42AGe18m4/edit#heading=h.ui5y06u3hly7), but for now I think this is totally fine.

I think we should aim for more using real values by Phase 2 or 3.

Comment (akostibas) [reply cmd/generate_test_data/main.go]: PS. I feel like a buzz kill :)

File: cmd/generate_test_data/main.go [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+		StandardCarrierAlphaCode: "ABCD",
+		Name: "Very Good TSP"}
+
+	tsp2 := models.TransportationServiceProvider{
+		StandardCarrierAlphaCode: "EFGH",
+		Name: "Pretty Alright TSP"}
+
+	tsp3 := models.TransportationServiceProvider{
+		StandardCarrierAlphaCode: "IJKL",
+		Name: "Serviceable and Adequate TSP"}
```
Comment (akostibas) [inline cmd/generate_test_data/main.go]: :)

File: cmd/generate_test_data/main.go (hunk 1 above)
Comment (breanneboland) [reply cmd/generate_test_data/main.go]: PR reviewers are supposed to be buzzkills, and it's feedback I welcome anyway.

File: cmd/generate_test_data/main.go (hunk 2 above)
Comment (breanneboland) [reply cmd/generate_test_data/main.go]: I'll KO this in the next (complementary) PR if it's still unneeded (and I think it will be).

**Comment by breanneboland** [comment]
Yaaaaay

**Pull request merged**